query-node/schema.graphql

```diff
@@ -83,6 +178,8 @@ type Channel @entity {
   # or a member. We are not handling events related to curator group so we will not set this field
   # owner: Member!
 
+  category: ChannelCategory
+
```

Comment:
handle is renamed to `title` and is not unique

Reply:
Done